ci-jobs/create-release-branch.yml

```diff
@@ -0,0 +1,35 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+pool:
+  vmImage: 'macOS 10.14'
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '12.x'
+- script: npm install
+  displayName: NPM install
+- script: |
+    git remote add triager https://triager:$GITHUB_TOKEN@github.com/appium/appium
+    git checkout -b $MINOR_BRANCH_NAME
+    git status
+    node ./ci-jobs/scripts/set-package-json-version
+    rm -rf package-lock.json
+    npm shrinkwrap
+    git add .
+    git commit -n -m "Set version to $(MINOR_BRANCH_NAME).0-rc.0"
```

Comment:
Any reason why we are setting `-n`?

Reply:
@umutuzgur just to speed it up. The commit only adds shrinkwrap and changes the version, so we know that it won't fail validation.

Reply:
Gotcha